fs/zbd_zenfs.h

```diff
@@ -42,6 +45,14 @@ struct zenfs_aio_ctx {
   int fd;
 };
 
+enum ZoneState {
```

Comment:
1. We should document this `ZoneState` to help users better understand the underlying state-changing mechanism.
2. I would prefer to create a separate `MetaZoneState` to clarify this